### ISSUE TITLE
Gen.int64 (Range.ExponentialBounded()) causes OverflowException

### ISSUE THREAD

**Issue by cmeeren** [hedgehogqa/fsharp-hedgehog#185]
```f#
[<Fact>]
let ``can create exponentially bounded int64`` () =
    Property.check <| property {
        let! _ = Gen.int64 (Range.exponentialBounded ())
        return true
    }
```

Causes the following exception:

`Message: System.OverflowException : Value was either too large or too small for an Int64.`

**Comment by cmeeren** [comment]
Any chance of fixing this?

**Comment by moodmosaic** [comment]
I may be able to look into it next week (traveling at the moment). Thanks for the heads up☝

**Comment by cmeeren** [comment]
It seems this also happens with `uint64`: https://github.com/cmeeren/fsharp-hedgehog-experimental/issues/32#issuecomment-505968571

**Comment by moodmosaic** [comment]
Fixed in https://github.com/hedgehogqa/fsharp-hedgehog/pull/188, released as Hedgehog 0.8.3 :tada:
